src/Database/PostgreSQL/PQTypes/Model/Check.hs

```diff
@@ -15,7 +15,7 @@ import Prelude
 data Check = Check {
   chkName      :: RawSQL ()
 , chkCondition :: RawSQL ()
-, chkValidated :: Bool -- ^ Set to 'True' if check is created as NOT VALID and
+, chkValidated :: Bool -- ^ Set to 'False' if check is created as NOT VALID and
                        -- not validated afterwards.
```

Comment:
`fkValidated` should be set to `False` if check is created as NOT VALID and left like this (for whatever reason), so the current description is imo alright.

Reply:
> fkValidated should be set to False if check is created as NOT VALID and left like this (for whatever reason)

OK, maybe change both comments to say this instead of current wording? I find it less confusing than the current one (with the double negative).